google/cloud/bigquery/enums.py

```diff
@@ -232,6 +232,19 @@ def _make_sql_scalars_enum():
 StandardSqlDataTypes = _make_sql_scalars_enum()
 
 
+class EntityTypes:
```

Comment:
I think we can make this a true enum like we did some of the other classes in this module.

```suggestion
class EntityTypes(str, enum.Enum):
```